containers/be-redis-pm2/Dockerfile

```
@@ -31,34 +31,40 @@ LABEL git="https://github.com/lehuygiang28/vphim"
 
 WORKDIR /usr/src/app
 
-# Install cloudflared
-RUN curl -L --output cloudflared https://github.com/cloudflare/cloudflared/releases/latest/download/cloudflared-linux-amd64 && \
-  chmod +x cloudflared && \
-  mv cloudflared /usr/local/bin/
+RUN npm install -g pm2
 
-# Install PM2
-RUN npm install -g pm2 dotenv-cli
+RUN chown node:node /usr/src/app
```

Hadolint

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)